containers/cuda/Containerfile

```diff
@@ -16,7 +15,12 @@ RUN export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cu
     && export XLA_FLAGS=--xla_gpu_cuda_data_dir=/usr/local/cuda
 
 ARG GIT_TAG=stable
-RUN CMAKE_ARGS="-DLLAMA_CUBLAS=on" CFLAGS="-mno-avx" python3.11 -m pip install -r https://raw.githubusercontent.com/instructlab/instructlab/${GIT_TAG}/requirements.txt --force-reinstall --no-cache-dir llama-cpp-python
+RUN if [ "$(uname -m)" = "aarch64" ]; then \
+        export CFLAGS=""; \
```

Comment:
Can't we just go with:

```bash
if [[ "$(uname -m)" != "aarch64" ]]; then
  export CFLAGS="-mno-avx"
fi
```

Or is `CFLAGS` always expected?j

Reply:
@leseb done! double checked on a graviton and amd64 instances.

Reply:
Thanks!